templates/publisher/publicise/github_badges.html

```diff
@@ -63,19 +60,14 @@ <h4>Promote your snap using embeddable GitHub badge</h4>
         <label>Markdown:</label>
       </div>
       <div class="col-7">
-        <div class="p-code-copyable not-cli">
-          <code class="p-code-copyable__input" id="snippet-badge-markdown">[![{{ snap_title }}](https://snapcraft.io/{{ snap_name }}/badge.svg)](https://snapcraft.io/{{ snap_name }})</code>
-          <button class="p-code-copyable__action js-clipboard-copy" data-clipboard-target="#snippet-badge-markdown">Copy to clipboard</button>
+        <div class="p-code-snippet">
+          <pre class="p-code-snippet__block is-wrapped" style="white-space: pre-wrap;"><code id="snippet-badge-markdown">[![{{ snap_title }}](https://snapcraft.io/{{ snap_name }}/badge.svg)](https://snapcraft.io/{{ snap_name }})</code></pre>
```

Comment:
Assuming `is-wrapped` would be added to SCSS, inline styles could be removed.
```suggestion
          <pre class="p-code-snippet__block is-wrapped"><code id="snippet-badge-markdown">[![{{ snap_title }}](https://snapcraft.io/{{ snap_name }}/badge.svg)](https://snapcraft.io/{{ snap_name }})</code></pre>
```